astroid/brain/brain_ssl.py

```diff
@@ -64,8 +68,101 @@ def ssl_transform():
     from _ssl import _OPENSSL_API_VERSION
     from _ssl import PROTOCOL_SSLv23, PROTOCOL_TLSv1, PROTOCOL_TLSv1_1, PROTOCOL_TLSv1_2
     from _ssl import PROTOCOL_TLS, PROTOCOL_TLS_CLIENT, PROTOCOL_TLS_SERVER
+
+    class AlertDescription(_IntEnum):
+        ALERT_DESCRIPTION_ACCESS_DENIED = 0
+        ALERT_DESCRIPTION_BAD_CERTIFICATE = 1
+        ALERT_DESCRIPTION_BAD_CERTIFICATE_HASH_VALUE = 2
+        ALERT_DESCRIPTION_BAD_CERTIFICATE_STATUS_RESPONSE = 3
+        ALERT_DESCRIPTION_BAD_RECORD_MAC = 4
+        ALERT_DESCRIPTION_CERTIFICATE_EXPIRED = 5
+        ALERT_DESCRIPTION_CERTIFICATE_REVOKED = 6
+        ALERT_DESCRIPTION_CERTIFICATE_UNKNOWN = 7
+        ALERT_DESCRIPTION_CERTIFICATE_UNOBTAINABLE = 8
+        ALERT_DESCRIPTION_CLOSE_NOTIFY = 9
+        ALERT_DESCRIPTION_DECODE_ERROR = 10
+        ALERT_DESCRIPTION_DECOMPRESSION_FAILURE = 11
+        ALERT_DESCRIPTION_DECRYPT_ERROR = 12
+        ALERT_DESCRIPTION_HANDSHAKE_FAILURE = 13
+        ALERT_DESCRIPTION_ILLEGAL_PARAMETER = 14
+        ALERT_DESCRIPTION_INSUFFICIENT_SECURITY = 15
+        ALERT_DESCRIPTION_INTERNAL_ERROR = 16
+        ALERT_DESCRIPTION_NO_RENEGOTIATION = 17
+        ALERT_DESCRIPTION_PROTOCOL_VERSION = 18
+        ALERT_DESCRIPTION_RECORD_OVERFLOW = 19
+        ALERT_DESCRIPTION_UNEXPECTED_MESSAGE = 20
+        ALERT_DESCRIPTION_UNKNOWN_CA = 21
+        ALERT_DESCRIPTION_UNKNOWN_PSK_IDENTITY = 22
+        ALERT_DESCRIPTION_UNRECOGNIZED_NAME = 23
+        ALERT_DESCRIPTION_UNSUPPORTED_CERTIFICATE = 24
+        ALERT_DESCRIPTION_UNSUPPORTED_EXTENSION = 25
+        ALERT_DESCRIPTION_USER_CANCELLED = 26
+
+    class SSLErrorNumber(_IntEnum):
+        SSL_ERROR_EOF = 0
+        SSL_ERROR_INVALID_ERROR_CODE = 1
+        SSL_ERROR_SSL = 2
+        SSL_ERROR_SYSCALL = 3
+        SSL_ERROR_WANT_CONNECT = 4
+        SSL_ERROR_WANT_READ = 5
+        SSL_ERROR_WANT_WRITE = 6
+        SSL_ERROR_WANT_X509_LOOKUP = 7
+        SSL_ERROR_ZERO_RETURN = 8
+
+    class VerifyMode(_IntEnum):
+        CERT_NONE = 0
+        CERT_OPTIONAL = 1
+        CERT_REQUIRED = 2
     """
-    )
+
+    def options_enum() -> str:
+        enum = """
+    class Options(_IntFlag):
+        OP_ALL = 1
+        OP_NO_SSLv2 = 2
+        OP_NO_SSLv3 = 3
+        OP_NO_TLSv1 = 4
+        OP_NO_TLSv1_1 = 5
+        OP_NO_TLSv1_2 = 6
+        OP_NO_TLSv1_3 = 7
+        OP_CIPHER_SERVER_PREFERENCE = 8
+        OP_SINGLE_DH_USE = 9
+        OP_SINGLE_ECDH_USE = 10
+        OP_NO_COMPRESSION = 11
+        OP_NO_TICKET = 12
+        OP_NO_RENEGOTIATION = 13
+        """
+
+        if PY38_PLUS:
+            enum += """
+        OP_ENABLE_MIDDLEBOX_COMPAT = 14
+        """
+
+        return enum
```

Comment:
```suggestion
        OP_NO_RENEGOTIATION = 13"""
        if PY38_PLUS:
            enum += """
        OP_ENABLE_MIDDLEBOX_COMPAT = 14"""
        return enum
```

This can be a bit more concise, similar to:

https://github.com/PyCQA/astroid/blob/20adb5420d29b7ad9528941c1eb391e58e004db3/astroid/brain/brain_collections.py#L61-L66